From e66af8cbd064e811591a6ef55f5280847bef87c1 Mon Sep 17 00:00:00 2001
From: Quanyang Wang <quanyang.wang@windriver.com>
Date: Wed, 10 Apr 2019 15:08:43 +0800
Subject: [PATCH 18/30] usb: dwc3: core: fix compile error

In upstream commit 44feb8e6ea2d ("usb: dwc3: use local copy of
resource to fix-up register offset"), err0 has been removed and
we need return value directly.

Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 drivers/usb/dwc3/core.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/dwc3/core.c b/drivers/usb/dwc3/core.c
index 231de8e2cd78..98077d9a67a0 100644
--- a/drivers/usb/dwc3/core.c
+++ b/drivers/usb/dwc3/core.c
@@ -1339,7 +1339,7 @@ static int dwc3_probe(struct platform_device *pdev)
 		ret = of_usb_update_otg_caps(dev->of_node,
 					&dwc->otg_caps);
 		if (ret)
-			goto err0;
+			return ret;
 	}
 
 	dwc->reset = devm_reset_control_get_optional_shared(dev, NULL);
-- 
2.17.1

